app/src/main/java/ru/flicksbox/data/AppError.kt

```diff
@@ -0,0 +1,5 @@
+package ru.flicksbox.data
+
+import java.io.IOException
+
+data class AppError(val userMessage: String) : IOException()
```

Comment:
I recommend to use more specific names for your own exceptions, and do not use common names
Also please use `Exception` postfix instead of `Error`